images/linux/Ubuntu2004-Readme.md

```diff
@@ -174,6 +174,7 @@
 - 2.1.302 2.1.403 2.1.526 2.1.617 2.1.701 2.1.818 3.1.120 3.1.202 3.1.302 3.1.416 5.0.104 5.0.210 5.0.303 5.0.404
 
 ### Databases
+- Cassandra 4.0.1 (apt source repository: https://downloads.apache.org/cassandra/)
```

Comment:
READMEs should not be updated manually, SoftwareReport does that

Reply:
Taken care of